### PR TITLE
added isPublished and publish object during upload and updateFileDetails respectively

### PR DESCRIPTION
**Pull request by ankur-dwivedi**

### CODE REVIEW

**Comment by ahnv** [comment]
Please bump the version

**Comment by ankur-dwivedi** [comment]
> Please bump the version



> Please bump the version

We are doing it here

**Comment by ahnv** [comment]
Please remove package-lock.json, we are using yarn.lock

**Comment by ankur-dwivedi** [comment]
> Please remove package-lock.json, we are using yarn.lock

done removed it

File: libs/manage/file.ts [hunk 1]
```diff
@@ -202,13 +202,20 @@ const updateDetails = function (
...
+      publish: updateData.publish,
+    };
+  else
+    data = {
+      tags: updateData.tags,
+      customCoordinates: updateData.customCoordinates,
+      extensions: updateData.extensions,
+      webhookUrl: updateData.webhookUrl,
+      customMetadata: updateData.customMetadata,
+    };
```
Comment (ahnv) [inline libs/manage/file.ts:218]: Similar to https://github.com/imagekit-developer/imagekit-python/pull/69/files#r1745669578

Comment (ankur-dwivedi) [reply libs/manage/file.ts]: Done fixed it

**Pull request merged**